plugins/modules/dhcp_option_space.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: dhcp_option_space
+short_description: Manage a OptionSpace
+description:
+    - Manages a DHCP Option Code by defining its properties, such as code, type, option space, and associated metadata, for DHCP configurations.
+version_added: 2.0.0
```

Comment:
```suggestion
version_added: 1.0.0
```

The corresponding generator change for this has been made !

Reply:
done